src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -200,9 +200,13 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
                 $queryBuilder->addSelect($associationAlias);
             }
 
-            // Avoid recursion
+            // Avoid recursion joins
```

Comment:
This is confusing... What are "recursion joins"?

Reply:
`Avoid recursive joins`, misspell :D